Dockerfile.QA

```diff
@@ -245,6 +246,10 @@ RUN cp client/src/grpc_generated/go/*.go qa/L0_simple_go_client/. && \
     cp client/src/grpc_generated/javascript/*.json qa/L0_simple_nodejs_client/. && \
     cp -r client/src/grpc_generated/java qa/L0_client_java/.
 
+# Copy Java API jar
+RUN if [ "$TARGETPLATFORM" = "linux/amd64" ]; then \
+        cp install/java-api-bindings/tritonserver-java-bindings.jar qa/pkgs/. ;\
```

Comment:
Nit: If the semicolon is necessary, is the formatting right? Not "qa/pkgs/.; \"?